BoxContentSDK/BoxContentSDK/QueueManagers/BOXParallelAPIQueueManager.m

```diff
@@ -51,6 +52,10 @@ - (id)initWithSession:(BOXAbstractSession *)session
         _uploadsQueue.name = @"BOXParallelAPIQueueManager upload queue";
         _uploadsQueue.maxConcurrentOperationCount = 2;
 
+        _thumbnailDownloadsQueue = [[NSOperationQueue alloc] init];
+        _thumbnailDownloadsQueue.name = @"BOXParallelAPIQueueManager thumbnails downloads queue";
+        _thumbnailDownloadsQueue.maxConcurrentOperationCount = 10;
```

Comment:
it's very hard to say. At some point between very good performance and very good performance, it's hard to see a difference. Those are very small operations data and even on a 3G connection, or at least how the network conditioner interprets a 3G connection, performances are still solid, downgrading the number of connection does not result in a noticeable change.
I'll bring it down to 8, which is the threshold where I stopped noticing performance differences.